src/lib/components/AssistantSettings.svelte

```diff
@@ -125,6 +135,12 @@
 					<span class="text-xs text-gray-500 hover:underline">Click to upload</span>
 				{/if}
 				<p class="text-xs text-red-500">{getError("avatar", form)}</p>
+				{#if (!files || !files[0]) && $page.data.avatarGeneration && !assistant?.avatar}
```

Comment:
```suggestion
				{#if !files?.[0] && $page.data.avatarGeneration && !assistant?.avatar}
```

I think `!files || !files[0])`, can be just `!files?.[0]`. I haven't tested it

Reply:
:bow:  it works pretty well